app/helpers/template_helpers.py

```diff
@@ -42,6 +42,90 @@ def get_page_header_context(language, theme):
     return context.get(theme)
 
 
+def get_footer_context(
```

Comment:
This method should have the same cache decorator (`@lru_cache(maxsize=None)`) as the `get_page_header_context` call. This ensures that the context is only ever generated once for any given combination of arguments.

Reply:
This causes an `unhashable type` error as the static urls are a dict.